check_modulemd.py

```diff
@@ -133,8 +133,12 @@ def test_description(self):
             "Checking for presence of description that is properly punctuated")
         if self.mmd.description and len(self.mmd.description) > 0:
             if not self.mmd.description.endswith('.'):
-                self.error("Description should end with a period: %s" %
-                           self.mmd.description)
+                if self.mmd.description.endswith('.\n'):
```

Comment:
My thought would be to warn if there's any trailing newline and/or whitespace found. Then, ignoring any such trailing newline/whitespace, just warn (rather than error) if there is no trailing period.